popSimManu.tex

```diff
@@ -399,36 +401,36 @@ \subsection*{Use case: comparing methods of demographic inference}
 \end{center}
 \end{figure}
 
-\paragraph*{Multi Population demographic models}
+\paragraph*{Multi-population demographic models}
 As \texttt{stdpopsim} implements multi-population demographic models, we also
 explored parameter estimation of population divergence parameters. In particular
 we performed simulations under multi-population models for humans and \emph{Drosophila}
 and inferred simulated parameters using \dadi, \fastsimcoal, and \smcpp.
-For simplicity, we conducted inference in \dadi and \fastsimcoal using an IM model
+For simplicity, we conducted inference in \dadi and \fastsimcoal by fitting an island migration (IM) model
```

Comment:
IM stands for isolation with migration model I think. Perhaps we should cite a Jody Hey paper for reference.

Reply:
i can add appropriate citations once this is merged.